client/scripts/views/sublayout.ts

```diff
@@ -57,14 +57,22 @@ const Sublayout: m.Component<{
       (!app.activeId() && !app.isCustomDomain() && (m.route.get() === '/' || m.route.get().startsWith('/?'))) ? [
         m('h3', 'Commonwealth')
       ] : chain ? [
-        m(ChainIcon, { size: ICON_SIZE, chain }),
+        m('.ChainIcon',[
+          link('a', (!app.isCustomDomain() ? `/${app.activeId()}` : app.customDomainId() ), [
```

Comment:
This should just be `/` for custom domains.

Reply:
Changed and pushed